src/Cli/dotnet/commands/dotnet-workload/list/WorkloadListCommand.cs

```diff
@@ -105,15 +105,12 @@ public override int Execute()
                     InstalledWorkloadsCollection installedWorkloads = _workloadListHelper.AddInstalledVsWorkloads(installedList);
                     PrintableTable<KeyValuePair<string, string>> table = new();
                     table.AddColumn(InformationStrings.WorkloadIdColumn, workload => workload.Key);
-                    if (!shouldShowWorkloadSetVersion)
+                    table.AddColumn(InformationStrings.WorkloadManifestVersionColumn, workload =>
                     {
-                        table.AddColumn(InformationStrings.WorkloadManifestVersionColumn, workload =>
-                        {
-                            var m = _workloadListHelper.WorkloadResolver.GetManifestFromWorkload(new WorkloadId(workload.Key));
-                            var manifestInfo = manifestInfoDict[m.Id];
-                            return m.Version + "/" + manifestInfo.ManifestFeatureBand;
-                        });
-                    }
+                        var m = _workloadListHelper.WorkloadResolver.GetManifestFromWorkload(new WorkloadId(workload.Key));
```

Comment:
Any chance this can through key not found exceptions or return null?

Reply:
That would mean we'd installed a workload but didn't install its associated manifests, which would mean we have a bug in the WorkloadInstallCommand.

Also, it's moved around a little, but this call has existed for years:
https://github.com/dotnet/sdk/blame/4acb5698e77cd7d6dfee3e4059faae94f6bbe7e8/src/Cli/dotnet/commands/dotnet-workload/list/WorkloadListCommand.cs#L90